www-servers/lighttpd/lighttpd-9999.ebuild

```
@@ -0,0 +1,251 @@
+		verify-sig? ( https://download.lighttpd.net/lighttpd/releases-$(ver_cut 1-2).x/${P}.tar.xz.asc )
+	"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+fi
+
+LICENSE="BSD GPL-2"
+SLOT="0"
+IUSE="+brotli crypto-gnutls crypto-mbedtls crypto-nettle crypto-openssl dbi gnutls kerberos ldap libdeflate +lua maxminddb mbedtls +nettle nss +pcre php sasl selinux ssl test unwind webdav xattr +zlib zstd"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="
+	lua? ( ${LUA_REQUIRED_USE} )
+"
+
+# Match the bundled xxhash version for the minimum version
+COMMON_DEPEND="
+	acct-group/lighttpd
+	acct-user/lighttpd
+	>=dev-libs/xxhash-0.8.2
+	virtual/libcrypt:=
+	brotli? ( app-arch/brotli:= )
+	crypto-gnutls? ( net-libs/gnutls )
+	crypto-mbedtls? ( net-libs/mbedtls )
+	crypto-nettle? ( dev-libs/nettle:= )
+	crypto-openssl? ( >=dev-libs/openssl-0.9.7:= )
+	dbi? (
+		dev-db/libdbi
+	)
+	gnutls? ( net-libs/gnutls )
+	kerberos? ( virtual/krb5 )
+	ldap? ( >=net-nds/openldap-2.1.26:= )
+	libdeflate? ( app-arch/libdeflate )
+	lua? ( ${LUA_DEPS} )
+	maxminddb? ( dev-libs/libmaxminddb )
+	mbedtls? ( net-libs/mbedtls )
+	nettle? ( dev-libs/nettle:= )
+	nss? ( dev-libs/nss )
+	pcre? ( dev-libs/libpcre2 )
+	php? ( dev-lang/php:*[cgi] )
+	sasl? ( dev-libs/cyrus-sasl )
+	ssl? ( >=dev-libs/openssl-0.9.7:= )
+	unwind? ( sys-libs/libunwind:= )
+	webdav? (
+		dev-libs/libxml2
+		dev-db/sqlite
+	)
+	xattr? ( kernel_linux? ( sys-apps/attr ) )
+	zlib? ( >=sys-libs/zlib-1.1 )
+	zstd? ( app-arch/zstd:= )
+"
+DEPEND="
+	${COMMON_DEPEND}
+	elibc_musl? ( sys-libs/queue-standalone )
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	selinux? ( sec-policy/selinux-apache )
+"
+BDEPEND="
+	virtual/pkgconfig
+	test? ( virtual/perl-Test-Harness )
+	verify-sig? ( sec-keys/openpgp-keys-lighttpd )
+"
+
+# update certain parts of lighttpd.conf based on conditionals
+update_config() {
+	local config="${ED}/etc/lighttpd/lighttpd.conf"
+
+	# Enable php/mod_fastcgi settings
+	if use php; then
+		sed -i -e 's|#.*\(include.*fastcgi.*$\)|\1|' ${config} || die
+	fi
+
+	# Automatically listen on IPv6 if built with USE=ipv6 (which we now always do)
+	# bug #234987
+	sed -i -e 's|# server.use-ipv6|server.use-ipv6|' ${config} || die
+}
+
+pkg_setup() {
+	if use lua; then
+		lua-single_pkg_setup
+	fi
+
+	if ! use pcre ; then
+		ewarn "It is highly recommended that you build ${PN}"
+		ewarn "with perl regular expressions support via USE=pcre."
+		ewarn "Otherwise you lose support for some core options such"
+		ewarn "as conditionals and modules such as mod_re{write,direct}."
+	fi
+
+	DOC_CONTENTS="IPv6 migration guide:\n
+		https://wiki.lighttpd.net/IPv6-Config
+	"
+}
+
+src_configure() {
+
+	local c_defs
+	local with_nettle
+
+	# choose crypto library
+	# (one specific library might be preferred on embedded systems via
+	#  MYMESONARGS with e.g. -DFORCE_blah_CRYPTO)
+	if use crypto-nettle ; then
+		with_nettle=$(meson_use crypto-nettle with_nettle)
+	else if use nettle ; then
```

I think this will fix the sourcing error:
```suggestion
	elif nettle ; then
```

---

doh!  thank you!  fixed.

---

That was it.  Incorrect shell syntax.  Sorry.  Thanks for diagnosing that.

The CI report is now clean for this PR.